app/main.py

```diff
@@ -17,10 +17,13 @@ def battle(knights_config: dict) -> dict:
         )
         knight.get_ready()
         knights_list.append(knight)
-    lancelot = knights_list[0]
-    arthur = knights_list[1]
-    mordred = knights_list[2]
-    red_knight = knights_list[3]
+    (lancelot,
+     arthur,
+     mordred,
+     red_knight) = (knights_list[0],
+                    knights_list[1],
+                    knights_list[2],
+                    knights_list[3])
```

Comment:
It isn't unpacking actually ;)
You can do it in this way
```
lancelot, arthur, mordred, red_knight = knights_list
```
If there are enough values, python will distribute values among knights, otherwise error will be raised